src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -63,7 +63,7 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
         String author = UserStore.getInstance()
           .getUser(message.getAuthorId()).getName();
     %>
-      <li><strong><%= author %>:</strong> <%= message.getContent() %></li>
+      <li><strong><a href="/profile/<%= author%>" style="text-decoration: none; color:black;"><%= author%>:</a></strong> <%= message.getContent() %></li>
```

Comment:
Consider doing the styling with CSS so it can be reused. In general it is best practice  to put styling in CSS instead of doing it inline. I know the existing code may not be consistent about this, but it is a good habit to get into. Also in this case having it in CSS will make it easier to add similar links on the activity page and elsewhere.